crates/sncast/tests/e2e/main_tests.rs

```diff
@@ -269,7 +269,7 @@ async fn test_keystore_declare() {
         .current_dir(contract_path.path())
         .args(args);
 
-    snapbox.assert().success().get_output().stderr.is_empty();
+    assert!(snapbox.assert().success().get_output().stderr.is_empty());
 
     fs::remove_dir_all(contract_path).unwrap();
```

Comment:
Not directly related to this PR, but I think `remove_dir_all` might not be necessary here. Since `contract_path` is a `TempDir`, it should be automatically deleted when it goes out of scope. What do you think?

Reply:
Definitely, good catch

Reply:
yep, probably someone copied from another test, which write to current_dir